app/main/posts/views/post-views.controller.js

```diff
@@ -4,14 +4,24 @@ PostViewsController.$inject = ['$scope', '$translate', '$transition$', 'PostFilt
 function PostViewsController($scope, $translate, $transition$, PostFilters) {
     // Set view and layout based out route
     $scope.currentView = $transition$.params().view;
+    $scope.transitionTo = $transition$.to().name;
```

Comment:
Is this reliable? ie. if I reload the view does it get set to the same thing again?

Reply:
yea. If you reload, you are still going to the same view and triggering the same transition. .from() can be weird, but to() should not be weird because you are still targeting the same "to "